### PR TITLE
feat(FTDAS): Create task entity and migrate

### PR DESCRIPTION
**Pull request by faysal-alam-sazim**
### Description of Change
- Created Task entity file 
- Create and up the migration to database
### Associated Ticket Link:
- https://trello.com/c/1GR78SZY
### Screenshots / Screen Recordings: N/A

### CODE REVIEW

File: src/common/entities/task.entity.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  @PrimaryKey()
+  id!: number;
+
+  @Property()
+  title!: string;
+
+  @Property({ nullable: true })
+  description?: string;
+
+  @Property({ nullable: true })
```
Comment (ishmam-iqbal-sazim) [inline src/common/entities/task.entity.ts:14]:
```suggestion
  @Property({ nullable: true, fieldName: "due_date" })
```
Please be explicit about the db fieldName in your entity file. All @Property decorator inside entity file should contain fieldName for all attributes except for id.

**Review by ishmam-iqbal-sazim — APPROVED** [review]
Approved pending comment

File: db/migrations/.snapshot-fancy-todo-app.json
Comment (ragib-hossain-sazim) [inline db/migrations/.snapshot-fancy-todo-app.json:1]: migration snapshot shouldn't be part of PR commit, add them in .gitIgnore file.

File: temp/Task.ts.json
Comment (ragib-hossain-sazim) [inline temp/Task.ts.json:1]: I think this temp folder files were generated for the metadatProvider that you are using. I believe it's not required to version them, you can put this folder inside .gitIgnore file.

**Review by ragib-hossain-sazim — APPROVED** [review]
Approved pending comments

**Pull request merged**